sound/soc/sof/topology.c

```diff
@@ -1256,15 +1256,45 @@ static int sof_connect_dai_widget(struct snd_soc_component *scomp,
 
 		switch (w->id) {
 		case snd_soc_dapm_dai_out:
-			for_each_rtd_cpu_dai(rtd, i, cpu_dai)
-				cpu_dai->capture_widget = w;
+			for_each_rtd_cpu_dai(rtd, i, cpu_dai) {
+				/*
+				 * We expect that DAI widget will define in
+				 * right order by topology, however, it is also
+				 * fine if it doesn't since all DAI widgets
+				 * will be enabled/disabled at the same time
```

Comment:
@bardliao better to just say "the order of DAI widgets defined in topology doesnt matter as all the CPU DAI's will be enabled/disabled simultaneously". same thing below